controls/ubuntu-18.04-server-cis-5.2.18.rb

```diff
@@ -89,14 +89,23 @@
   "
   impact 0.5
   tag severity: "medium"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
   tag nist: ["AC-6 (9)", "Rev_4"]
   tag cis_level: 1
   tag cis_controls: ["4.3", "Rev_7"]
   tag cis_rid: "5.2.18"
+
+  describe.one do
+    describe parse_config_file('/etc/ssh/sshd_config', { assignment_regex: /^\s*(\S*)\s+(.*?)\s*$/ } ) do
+      its('AllowUsers') { should be }
```

Comment:
it should be what?

Reply:
The control here specifies that at least one of these is defined.